tests/state_test.go

```diff
@@ -50,12 +51,10 @@ func TestState(t *testing.T) {
 		st.skipLoad(`^stEWASM`)
 	}
 	if *testEVM != "" {
-
-		// These fail because these forks were not supported at this version of the EVMOne .so.
-		// The EVMOne version (0.2.0) is the latest EVMC v6 compatible version.
-		st.skipFork("Constantinople") // Only support
-		st.skipFork("Istanbul")
-		st.skipFork("ETC_Phoenix")
+		// These interpreters fail Constantinople (but pass ConstantinopleFix).
+		if strings.Contains(*testEVM, "aleth") || strings.Contains(*testEVM, "evmone") {
+			st.skipFork("^Constantinople$")
```

Comment:
:eyes: 

I haven't looked at why these fail (nor at if they're even expected to pass) (the symptom is the typical `mismatched post state root`). 

Since Constantinople was (and is) followed by `ConstantinopleFix` (aka `Petersbug`) I'm inclined to consider the `Constantinople` fork as an ugly duckling and allow it to fail here without further pursuit. From its disuse (at least relative to ETH and ETC), I think it's probably fair to consider it "poorly supported"... at least as far as the the EVM domain goes (and from our current time onward).

Reply:
I'm ok with this line of thinking. Other than the initial foundation testnet, no networks should be activating Constantinople and petersburg separately.